src/pages/VideoListing.js

```diff
@@ -0,0 +1,65 @@
+import axios from 'axios'
+import React, { useEffect, useState } from 'react'
+import { VideoCard } from '../components'
+import { SearchCard } from '../components/Search/Search'
+import { errorToast } from '../components/Toast/Toast'
+import { DATA_FROM_SERVER } from '../constants/reducerConstants'
+import { useVideo } from '../context/VideoContext'
+export const VideoListing = () => {
+  const [isLoading, setIsLoading] = useState(false)
+  const { videoState, videoDispatch } = useVideo()
+  const searchResult = (videos, search) => {
+    if (search) {
+      videos = [...videos].filter((video) =>
+        video.title.toLowerCase().includes(search)
```

Comment:
When we use the filter method, we don't need to destructure the array on which it is called. Filter will not mutate the original array

Reply:
Oh yes, I have missed this 
Thank you